models/sync-media/README.md

```diff
@@ -0,0 +1,103 @@
+# Synchronized Media
+
+This document introduces a new JSON based syntax for synchronizing media together.
+
+## Listing documents
+
+
+```json
+"syncMedia": [
+  {
+    "href": "sync.json"
+  },
+  {
+    "href": "sync2.json"
+  }
+]
+```
+
+## Model
+
+The new syntax is based around a single JSON object called a Sync Media Object:
+
+| Name | Description | Format |
+| ---- | ----------- | ------ |
+| `audioref` | Points to a media fragment in an audio resource. | URI |
+| `children` | Array of Sync Media Objects. | Sync Media Objects |
+| `description` | Text, audio or image description for the current Sync Media Object. | Object |
+| `imgref` | Points to a media fragment in an image resource. | URI |
+| `role`     | Array of roles relevant for the current Sync Media Object. | Array of roles |
+| `text`  | Text equivalent for the current Sync Media Object. | String |
+| `textref`  | Points to a media fragment in an HTML/XHTML resource. | URI |
+
+Each Sync Media Object MUST contain:
+
+- a `children` object containg at least one Sync Media Object
+- or at least two of the following elements: `audioref`, `imgref`, `text` or `textref`
```

Comment:
There's only one element in the case of a resource without any associated media (e.g. bubble without text).

Reply:
Yeah, looks like we'll need to drop that requirement.

Based on the current state of things, looks like we'll need to request the presence of at least one of the following elements: `audioref`, `imgref` or `textref`.